apps/call.go

```diff
@@ -11,7 +11,7 @@ type CallType string
 
 const (
 	// CallTypeSubmit (default) indicates the intent to take action.
-	CallTypeSubmit = CallType("")
+	CallTypeSubmit = CallType("submit")
```

Comment:
If we want to default, I would do it at proxy level. My approach would be:
- Assume calls from the client are well formed (the client is our code and should be correct)
- Assume responses from the apps may be wrong. Default to OK if no type is received.
- Send always proper responses to the client.

We could add some defaulting in the trip client->proxy->app to be safe, but one part of me want to leave it like that to find errors ASAP.

Reply:
Response type is now defaulted to `ok`. I agree with no defaulting from `client->proxy->app`, to catch errors early.